controllers/provider-vsphere/example/old/30-controlplane.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: shoot--foobar--vsphere
+  labels:
+#    backup.gardener.cloud/provider: vsphere
+#    seed.gardener.cloud/provider: vsphere
+    shoot.gardener.cloud/provider: vsphere
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloudprovider
+  namespace: shoot--foobar--vsphere
+type: Opaque
+data:
+  apiToken: YXBpVG9rZW4K # apiToken
+  projectID: cHJvamVjdElkCg== # projectId
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Cluster
+metadata:
+  name: shoot--foobar--vsphere
+spec:
+  cloudProfile:
+    apiVersion: core.gardener.cloud/v1alpha1
```

Comment:
Can you change every occurrence of `core.gardener.cloud/v1alpha1` to `core.gardener.cloud/v1beta1`, please?